src/composables/useFlawAffectsModel.ts

```diff
@@ -12,13 +12,26 @@ import { useToastStore } from '@/stores/ToastStore';
 import type { ZodFlawType } from '@/types/zodFlaw';
 import type { ZodAffectType, ZodAffectCVSSType } from '@/types/zodAffect';
 import { deepCopyFromRaw } from '@/utils/helpers';
-import { equals } from 'ramda';
+import { sortWith, ascend, prop, equals } from 'ramda';
+
+
+const sortAffects = (affects: ZodAffectType[] = []) => {
+  return sortWith(
+    [
+      // get product value for Ramda ascend function
```

Comment:
I'd sightly modify this to also inform about what the function does.
E.g. "get product value using Ramda ascend function as comparator"